cc-pk-service/Dockerfile

```diff
@@ -0,0 +1,33 @@
+# This is where you define the image for
+# Materiaalitori service setup.
+
+# Start by copying the server stub to this directory.
+
+
+# First line is provided for your convenience.
+# This line fetches the Node Docker image with tag "16"
+# from Docker Hub.
+# We use version 16 as the mongodb dependency has requirements
+FROM node:latest
```

Comment:
The comment above does not reflect this line. Should we use some fixed version of node? Also we don't have mongodb

Reply:
updated